node/src/main/scala/org/bitcoins/node/networking/peer/PeerMessageSender.scala

```diff
@@ -198,7 +198,7 @@ case class PeerMessageSender(client: P2PClient)(implicit conf: NodeAppConfig)
       res <- filterSyncMarkerOpt match {
         case Some(filterSyncMarker) =>
           logger.info(
-            s"Requesting compact filters from=${filterSyncMarker.startHeight} to=${filterSyncMarker.stopBlockHash}")
+            s"Requesting compact filters from=${filterSyncMarker.startHeight} to=${filterSyncMarker.stopBlockHash.flip.hex}")
```

Comment:
Maybe we should just re-work the `FilterSyncMarker.toString`? and get rid of `filterSyncMarker.startheight` and `filterSyncMakrer.stopBlockHash` here and in the log below?